assets/javascript/apps/pipeline/PipelineNode.tsx

```diff
@@ -238,7 +238,12 @@ export function PipelineNode({ id, data, selected }: NodeProps<NodeData>) {
       { length: numberOfOutputs },
       (_, index) => {
         const position = (index / (numberOfOutputs - 1)) * 100; // Distributes evenly between 0% to 100%
-        const handleText = `Output ${index + 1}`;
+        let handleAnnotation;
```

Comment:
It would be nice if the widgets can optionally output where _it_ would like its output handle to be (if it is tied to one). This way each widget can optionally output a handle position, equal to its own Y position. We can then simply iterate over the list of output position requests and put the handles there. This way we can show the handles inline with the widget. I am not yet convinved that this is worth undertaking though, but just a thought